src/utils/read-pacts.ts

```diff
@@ -2,7 +2,13 @@ import * as fs from 'fs';
 import * as path from 'path';
 
 export function readPacts(dir: string): Pact[] {
-    return fs.readdirSync(dir).map((fileName) => {
+    const fileNames = fs.readdirSync(dir);
```

Comment:
We should also check if `dir` exists and throw an error if not

Reply:
It is already handled by readDirSync, error is thrown and caught in index.ts